### PR TITLE
Update calc.py

### PR DESCRIPTION
**Pull request by KennyOliver**
[+] Show the calculation options in a neater way
[+] Print the equations and answers together for an easier to understand UI

### CODE REVIEW

**Comment by PrajaktaSathe** [comment]
I checked the code. Good work on the UI here! 
Just one change I would suggest is that we should keep the option of continuing calculations for the user. In your program only one calculation can be performed, after which the program ends. Could please make it so that the user can choose when to stop? Thanks!

**Comment by KennyOliver** [comment]
Hi! Thank you so much for the feedback; it’s motivating! 😄
I’ve fixed the issue & added extra stuff.

**Review by PrajaktaSathe — APPROVED** [review]
Thanks for updating!! Good work here :smile:

**Comment by PrajaktaSathe** [comment]
> Hi! Thank you so much for the feedback; it’s motivating! 😄
> I’ve fixed the issue & added extra stuff.

Thank you!! 😃

File: Projects/calc.py [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+        num1 = float(input("1st number --> "))
+        num2 = float(input("2nd number --> "))
+        print(f"{num1} - {num2} = {num1 - num2}")
+    elif ch == 3: #multiplication
+        num1 = float(input("1st number --> "))
+        num2 = float(input("2nd number --> "))
+        print(f"{num1} x {num2} = {num1 * num2}")
+    elif ch == 4: #division
+        num1 = float(input("Dividend --> "))
+        num2 = float(input("Divisor  --> "))
```
Comment (PrajaktaSathe) [inline Projects/calc.py:19]: @KennyOliver 
Could you please add one more correction? Dividing 0 by 0 should not be considered valid. So make the program display an error message when the user enters 0 as the divisor.

Comment (KennyOliver) [reply Projects/calc.py:19]: Sure, I’ve just made a new PR.

**Pull request merged**